app/_src/production/deployment/multi-zone.md

```diff
@@ -0,0 +1,96 @@
+
+In the presence of a [zone egress](/docs/{{ page.version }}/explore/zoneegress), the traffic is routed through the local zone egress before being sent to the remote zone ingress.
+
+When using [transparent proxy](/docs/{{ page.version }}/networking/transparent-proxying) (default in Kubernetes),
+{{site.mesh_product_name}} generates a VIP,
+a DNS entry with the format `<kuma.io/service>.mesh`, and will listen for traffic on port 80. The `<kuma.io/service>.mesh:80` format is just a convention.
+[`VirtualOutbounds`](/docs/{{ page.version }}/policies/virtual-outbound)s enable you to customize the listening port and how the DNS name for these services looks.
+
+{% tip %}
+A zone ingress is not an API gateway. It is only used for cross-zone communication within a mesh. API gateways are supported in {{site.mesh_product_name}} [gateway mode](/docs/{{ page.version }}/explore/gateway) and can be deployed in addition to zone ingresses.
+{% endtip %}
+
+### Components of a multi-zone deployment
+
+A multi-zone deployment includes:
+
+- The **global control plane**:
+  - Accept connections only from zone control planes.
+  - Accept creation and changes to [policies](/policies) that will be applied to the data plane proxies.
+  - Send policies down to zone control planes.
+  - Send zone ingresses down to zone control plane.
+  - Keep an inventory of all data plane proxies running in all zones (this is only done for observability but is not required for operations).
+  - Reject connections from data plane proxies.
+- The **zone control planes**:
+  - Accept connections from data plane proxies started within this zone.
+  - Receive policy updates from the global control plane.
+  - Send data plane proxies and zone ingress changes to the global control plane.
+  - Compute and send configurations using XDS to the local data plane proxies.
+  - Update list of services which exist in the zone in the zone ingress.
+  - Reject policy changes that do not come from global.
+- The **data plane proxies**:
+  - Connect to the local zone control plane.
+  - Receive configurations using XDS from the local zone control plane.
+  - Connect to other local data plane proxies.
+  - Connect to zone ingresses for sending cross zone traffic.
+  - Receive traffic from local data plane proxies and local zone ingresses.
+- The **zone ingress**:
+  - Receive XDS configuration from the local zone control plane.
+  - Proxy traffic from other zone data plane proxies to local data plane proxies.
+- (optional) The **zone egress**:
+  - Receive XDS configuration from the local zone control plane.
+  - Proxy traffic from local data plane proxies:
+    - to zone ingress proxies from other zones;
+    - to external services from local zone;
+
+{% if_version gte:2.2.x %}
+
+## More information
+
+For more information about how to configure a multi-zone deployment, see [Deploy a multi-zone global control plane](/docs/{{ page.version }}/production/cp-deployment/multi-zone/).
```

Comment:
@fabianrbz Do you know why this link is failing the link checker? It looks like it's only failing for 2.1, so I made this only appear for 2.2 (because that's when I made this doc change), but it's still failing.